src/vitest.setup.ts

```diff
@@ -0,0 +1,37 @@
+import * as matchers from '@testing-library/jest-dom/matchers'
+import { cleanup } from '@testing-library/react'
+import { vi } from 'vitest'
+import '@testing-library/jest-dom/vitest'
+
+// not sure why this lint is being fired here so I'm disabling it
+// eslint-disable-next-line testing-library/await-fire-event
+expect.extend(matchers)
+
+// Prevent timezone differences between local and CI/CD
+const setupTestGlobal = async () => {
+  process.env.TZ = 'UTC'
+}
+
+export default setupTestGlobal
+
+process.env.REACT_APP_ZOD_IGNORE_TESTS = 'true'
```

Comment:
what does this do?

Reply:
Copied over from the Jest test setups ... something I believe @ajay-sentry added in.